processing-service-core/global-config-options.md

```diff
@@ -18,13 +20,14 @@ The body of the request would then contain the strings *default* and *descriptio
 
```

Comment:
Should mention scope, "The body of the request would then contain the strings *default*,*description* and *scope*".